dev/stats/get_important_pr_candidates.py

```diff
@@ -282,7 +287,10 @@ def score(self):
         # Weight PRs with protm tags more heavily:
         # If there is at least one protm tag, multiply the interaction score by the number of tags, up to 3.
         interaction_score = self.interaction_score
-        interaction_score *= min(self.protm_score + 1, 3)
+        try:
+            interaction_score *= min(self.protm_score + 1, 3)
+        except AttributeError:
```

Comment:
Why would this ever raise? Isn't `protm_score` set in init?

Reply:
@jedcunningham  I was puzzled, too, but it did when I was loading PRs from a Pickle file. I could remove it until it happens again...

Reply:
This can happen if the pickle is stale, but I think we should let it raise and resolve by re-building the pickle instead.